### DISCUSSION TITLE
Data Importe: pimcore.plugin.pimcoreDataImporterBundle.configuration.components.logTab is not a constructor

### DISCUSSION THREAD

**Discussion by adi84900** [pimcore/pimcore#15342] [Q&A / Support]
Hi All,

I installed the datahub and data importer in the community version. When I am creating an data object and trying to open the setting I am getting the below error. Can someone take a look?

pimcore.plugin.pimcoreDataImporterBundle.configuration.components.logTab is not a constructor

![image](https://github.com/pimcore/pimcore/assets/32416123/e635c3eb-b9cb-48f2-b198-e9d7bf1c1810)

@fashxp @willers-pm @NiklasBr @rajesh17692
Could you please help here

**Comment by dvesh3** [comment]
Issue resolved here https://github.com/pimcore/data-importer/issues/344
